tar.go

```diff
@@ -51,6 +51,8 @@ type Tar struct {
 	readerWrapFn  func(io.Reader) (io.Reader, error)
 	writerWrapFn  func(io.Writer) (io.Writer, error)
 	cleanupWrapFn func()
+
+	destination string
```

Comment:
I bet we can find a way to implement this without adding more state.

Reply:
Yeah, for sure. I just made it in the most bruteforce way possible without having to pass around empty strings to methods